packages/ember-meta/lib/meta.ts

```diff
@@ -390,67 +391,75 @@ export class Meta {
   }
 
   writeBindings(subkey: string, value: any) {
```

Comment:
maybe conditionally extend meta like done here: https://github.com/emberjs/ember.js/pull/16355

Reply:
would you mind updating that PR to fix it up? I do agree that is nice